app/features/utils/openExternalLink.js

```diff
@@ -2,7 +2,7 @@ const { shell } = require('electron');
 const url = require('url');
 
 
-const protocolRegex = /^https?:/i;
+const protocolRegex = /^(https|mailto)?:/i;
```

Comment:
```suggestion
const protocolRegex = /^(https?|mailto):/i;
```

Untested, but isn't this what we want?

Reply:
Yeah your right this is what we want. Made a typo.